docs/docker-image-details.md

```diff
@@ -49,12 +48,15 @@ You can deploy the standalone Gravitino Iceberg REST server with the Docker imag
 Container startup commands
 
 ```shell
-docker run --rm -d -p 9001:9001 datastrato/gravitino-iceberg-rest-server
+docker run --rm -d -p 9001:9001 apache/gravitino-iceberg-rest-server:0.6.0-incubating
 ```
 
 Changelog
 
-- gravitino-iceberg-rest-server:0.6.0
+- apache/gravitino-iceberg-rest-server:0.6.0-incubating (Switch to Apache official DockerHub repository)
+  - Use `datastrato/gravitino-iceberg-rest-server:0.6.0` Dockerfile to rebuild the image.
+
+- datastrato/gravitino-iceberg-rest-server:0.6.0
   - Docker image `datastrato/gravitino-iceberg-rest-server:0.6.0`
   - Gravitino Iceberg REST Server with memory catalog backend.
   - Expose ports:
```

Comment:
I'm curios why do you still keep this, there's no 0.6.0 release in official release, it will always be 0.6.0-incubating.

Reply:
I will remove the following lines if it's not proper to keep the version here as `0.6.0-incubating` is the latest version and only this version take effect. 
```
- datastrato/gravitino-iceberg-rest-server:0.6.0
  - Docker image `datastrato/gravitino-iceberg-rest-server:0.6.0`
  - Gravitino Iceberg REST Server with memory catalog backend.
  - Expose ports:
```